entity-types/ext-database/summary_metrics.yml

```diff
@@ -0,0 +1,16 @@
+state:
+  goldenMetric: state
+  title: Database State
+  unit: COUNT
+totalConnections:
+  goldenMetric: totalConnections
+  title: Total Connections
+  unit: COUNT
+size:
+  goldenMetric: size
+  title: Database Size in MB
+  unit: COUNT
```

Comment:
I don't think it's a count, I would remove the `in MB`  from the title and use unit BYTES  (multiplying the query * 1000 if needed)
Using the right unit the UI should do it's best to choose the most meaningful scale.

Reply:
switched to BYTES